eng/pipelines/official.yml

```diff
@@ -134,6 +134,26 @@ extends:
                 inputs:
                   artifactName: Artifacts_Windows_NT_Release
                   targetPath: $(Pipeline.Workspace)/Artifacts_Windows_NT_Release
-              - template: /eng/pipelines/templates/steps/workload-publish.yml@self
-                parameters:
-                  feedForPublishing: ${{ parameters.feedForPublishing }}
\ No newline at end of file
+              - task: 1ES.PublishNuget@1
+                condition: ${{ ne(parameters.feedForPublishing, '') }}
+                displayName: 🟣 Publish package to AzDO
+                inputs:
+                  useDotNetTask: true
+                  packagesToPush: $(Pipeline.Workspace)/Artifacts_Windows_NT_Release/packages/Release/Shipping/*.nupkg
+                  packageParentPath: $(Pipeline.Workspace)/Artifacts_Windows_NT_Release/
+                  publishVstsFeed: ${{ parameters.feedForPublishing }}
+                  nuGetFeedType: internal
+                  allowPackageConflicts: false
+                  publishPackageMetadata: true
+              - task: 1ES.PublishNuget@1
```

Comment:
Do we want a single approval for both AzDO and NuGet, or do we want separate approvals?
Are there cases where we want to publish to only one?